src/main/kotlin/no/nav/faktureringskomponenten/service/beregning/BeløpBeregner.kt

```diff
@@ -12,12 +12,10 @@ class BeløpBeregner {
     companion object {
         fun beløpForPeriode(enhetspris: BigDecimal, fom: LocalDate, tom: LocalDate): BigDecimal {
             val angittAntall = AntallBeregner(fom, tom).beregn()
-            val beløp = enhetspris.multipliserMedHeltall(angittAntall)
+            val beløp = enhetspris.multiply(angittAntall).setScale(2, RoundingMode.DOWN)
```

Comment:
Hvorfor var setScale nødvendig? Med to desimaler for angittAntall og ingen desimaler for enhetspris, burde vi ikke la beløp være slik det er? 

Vi kan jo ikke funksjonelt avrunde her fordi da fungerer ikke oracle, burde vi ikke da heller kaste feil eller warning om den har annet enn to desimaler?